CPTSuperLearn/interpolator.py

```diff
@@ -24,6 +26,33 @@ def predict(self, prediction_points: List[float]):
         raise NotImplementedError("Predict method must be implemented")
 
 
+class SchemaGANInterpolator(InterpolatorAbc):

```

Comment:
missing docstring in this class and in the functions of the class

Reply:
added